### DISCUSSION TITLE
Should C# Debugger settings changes be considered when restarting a debug session?

### DISCUSSION THREAD

**Discussion by mauroa** [OmniSharp/omnisharp-vscode#6910] [Q&A] [answered]
I noticed that If I change a C# Debugger setting like "Just My Code" in the middle of a debug session, when I restart the debug session (Ctrl + Shift + F5), the change is not applied until it's stopped and started again. Is it an expected behavior?

**Comment by gregg-miskelly** [comment, marked as answer]
That definitely sounds like a good feature, but, as you discovered, it doesn't work like that today. Would you mind opening a feature request issue?

**Reply by mauroa** [reply]
Done: https://github.com/dotnet/vscode-csharp/issues/6911
